### PR TITLE
#167891587 Reject Request (By Requester’s Manager)

### PR DESCRIPTION
**Pull request by Pomile**
#### What does this PR do?
Have a manager reject travel request

#### Description of Task to be completed?
- Reuse verify token middleware
- Create a permission middleware
- Create validation for trip request status
- Add method that modify trip request status

##### Unit test
1. Clone the project repository
2. Cd into the project repository
3. Run `git pull`
4. Run `git checkout  ft-update-travelRequestStatus-endpoint-167891587`
5. Run `npm install`
6. Create database `database_name`
7. Create .env file
8. Run npm test
##### Development test with Postman
 1. Run `npm run devstart`
 2. Launch postman
 3. Create a new patch request with  `/trips/:id` - `/api/v1/trips/:tripId` 
 4. Add a request body:  `{ "status": "rejected" }`
 5. Add authorization(token) in the request header.
 6. Hit send

#### What are the relevant pivotal tracker stories?
[#167891587](https://www.pivotaltracker.com/story/show/167891587)
#### Any background context you want to add?
Nil
#### Screenshots
modifyTripStatus

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -36,4 +53,4 @@
       }
     ]
   }
-}
+}
```
Comment (thatkazuk1) [inline .eslintrc]: Newline here

File: src/controllers/Travel.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import db from '../database/models';
```
Comment (thatkazuk1) [inline src/controllers/Travel.js]: I believe we agreed on naming our file `Trip`

File: src/routes/trip.routes.js [hunk 3]
```diff
@@ -2,16 +2,32 @@ import { Router } from 'express';
...
 tripRoutes.get('/user', Token.verifyToken, TripController.getUserTrips);
+
 tripRoutes.post(
   '/',
   Token.verifyToken,
   validator(oneWaySchema),
   TripController.createTripRequest
 );
 
+tripRoutes.get(
```
Comment (max-wel) [inline src/routes/trip.routes.js]: This route is the same as line 10

Comment (Pomile) [reply src/routes/trip.routes.js]: Thanks Max.
It is done.

**Review by daniellamarr — APPROVED** [review]
Nice work!

**Pull request merged**